### PR TITLE
affected route names

### PR DESCRIPTION
**Pull request by VysotskiVadim**
A new field in the [incident object](https://docs.mapbox.com/api/navigation/directions/#incident-object)

### CODE REVIEW

File: services-directions-models/src/main/java/com/mapbox/api/directions/v5/models/Incident.java [hunk 1]
```diff
@@ -466,6 +473,12 @@ public abstract static class Builder extends DirectionsJsonObject.Builder<Builde
      */
     public abstract Builder numLanesBlocked(@Nullable Integer numLanesBlocked);
 
+    /**
+     * Sets list of roads names affected by the incident.
+     * @param affectedRoadNames list of roads names affected by the incident.
+     */
+    public abstract Builder affectedRoadNames(@Nullable List<String> affectedRoadNames);
```
Comment (Guardiola31337) [inline services-directions-models/src/main/java/com/mapbox/api/directions/v5/models/Incident.java:480]:
```suggestion
     */
     @NonNull
    public abstract Builder affectedRoadNames(@Nullable List<String> affectedRoadNames);
```

We can also add the above missing annotations 👍

File: services-directions-models/src/test/java/com/mapbox/api/directions/v5/models/IncidentTest.java [hunk 2]
```diff
@@ -17,31 +17,21 @@ public void sanity(){
...
-      .countryCodeAlpha2("US")
-      .countryCodeAlpha3("USA")
-      .build();
+    Incident incident = getFilledIncident();
     byte[] serialized = TestUtils.serialize(incident);
     assertEquals(incident, deserialize(serialized, Incident.class));
   }
 
+  @Test
+  public void testToAndFromJson() {
```
Comment (Guardiola31337) [inline services-directions-models/src/test/java/com/mapbox/api/directions/v5/models/IncidentTest.java:26]: We should also remove `test` prefix from tests names, it's implicit with the usage of `@Test` annotation.

Same thing applies to other tests above and below.

**Review by Guardiola31337 — COMMENTED** [review]
Left a couple of minor comments.

File: services-directions-models/src/test/java/com/mapbox/api/directions/v5/models/IncidentTest.java (hunk 2 above)
Comment (VysotskiVadim) [reply services-directions-models/src/test/java/com/mapbox/api/directions/v5/models/IncidentTest.java]: I don't see anything bad in having a `test` word in the beginning in a test name. But if you have strong opinion about this, I don't mind changing the names.
Updated.

**Comment by RingerJK** [comment]
please extend `RouteLegRefreshTest` and add  `affected_road_names` to `directions_refresh_v1.json`, `directions_refresh_v1_obj.json`,  `directions_refresh_v1_obj_passed_first_leg.json` where it needs

**Review by RingerJK — APPROVED** [review]

**Pull request merged**